File: cloud-reactor/src/main/java/com/sequenceiq/cloudbreak/cloud/handler/DownscaleStackExecuter.java
package com.sequenceiq.cloudbreak.cloud.handler;

import com.sequenceiq.cloudbreak.cloud.event.resource.DownscaleStackRequest;
import com.sequenceiq.cloudbreak.cloud.event.resource.DownscaleStackResult;

interface DownscaleStackExecuter {
    DownscaleStackResult execute(DownscaleStackRequest request);
}

File: cloud-reactor/src/main/java/com/sequenceiq/cloudbreak/cloud/handler/DownscaleStackExecuter.kt
package com.sequenceiq.cloudbreak.cloud.handler

import com.sequenceiq.cloudbreak.cloud.event.resource.DownscaleStackRequest
import com.sequenceiq.cloudbreak.cloud.event.resource.DownscaleStackResult

internal interface DownscaleStackExecuter {
    fun execute(request: DownscaleStackRequest<Any>): DownscaleStackResult
}
